kivy/config.py

```diff
@@ -805,6 +811,14 @@ def name(self, value):
         elif version == 15:
             Config.setdefault('kivy', 'kivy_clock', 'default')
 
+        elif version == 16:
+            Config.setdefault('kivy', 'default_font', [
```

Comment:
Sorry, we do it internally; https://github.com/kivy/kivy/blob/master/kivy/config.py#L433.